src/test/suite/fixtures/recorded/selectionTypes/bringHarpToAfterFile.yml

```diff
@@ -0,0 +1,37 @@
+languageId: plaintext
```

Comment:
isn't this one identical to `src/test/suite/fixtures/recorded/positions/bringHarpToAfterFile.yml`?